homeassistant/components/unifi/button.py

```diff
@@ -91,6 +112,19 @@ class UnifiButtonEntityDescription(
         supported_fn=lambda hub, obj_id: bool(hub.api.ports[obj_id].port_poe),
         unique_id_fn=lambda hub, obj_id: f"power_cycle-{obj_id}",
     ),
+    UnifiButtonEntityDescription[Wlans, Wlan](
+        key="WLAN Change Password",
+        device_class=ButtonDeviceClass.UPDATE,
+        entity_category=EntityCategory.CONFIG,
+        entity_registry_enabled_default=False,
+        api_handler_fn=lambda api: api.wlans,
+        available_fn=async_wlan_available_fn,
+        control_fn=async_change_password_control_fn,
+        device_info_fn=async_wlan_device_info_fn,
+        name_fn=lambda wlan: "Change Password",
```

Comment:
```suggestion
        name_fn=lambda wlan: "Regenerate Password",
```
I think this is more explicit, change sounds like something that can take an input